### PR TITLE
Add email form validation

### PR DESCRIPTION
**Pull request by collinsmezie**
Javascript validation of the email input field

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi @collinsmezie, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have [JavaScript linter](https://github.com/microverseinc/linters-config/tree/master/javascript) Please add it and re-submit.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

File: index.js [hunk 1]
```diff
@@ -71,75 +71,98 @@ const data = [
...
+
+const form = document.getElementById("html-form");
+const email = document.getElementById("email");
+const message = document.getElementById("message");
+
+form.addEventListener("submit", (event) => {
+
+    if(email.value !== email.value.toLowerCase()){
+        event.preventDefault()
+        message.textContent = "EMAIL should be in lowercase";
```
Comment (gemmen29) [inline index.js:155]: Kindly change the error message of the email input position to be near the submit button instead. As it's in the requirement of the project.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @collinsmezie, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @collinsmezie, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**